### PR TITLE
Create utility to generate reports for adversarial queries.

### PR DESCRIPTION
**Pull request by keyurva**
The following will be implemented in subsequent PRs:
* Add "blocked" and debug["counter"]["INFO"]["info_palm_api_response"].
* Backoff and retry on palm 429s.

### CODE REVIEW

File: server/integration_tests/standalone/adversarial.py [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+    cosine_scores = sv_matching.get("CosineScore", [])
+    svs = sv_matching.get("SV", [])
+    if cosine_scores and svs:
+      sv_scores = []
+      for sv, score in zip(svs, cosine_scores):
+        if sv in svs_used:
+          sv_scores.append(f"{sv} = {score}")
+      result.sv_matching = "\n".join(sv_scores)
+
+  # set status
```
Comment (pradh) [inline server/integration_tests/standalone/adversarial.py:387]: Add "blocked" and debug["counter"]["INFO"]["info_palm_api_response"].

The latter is the raw response from palm.

File: server/integration_tests/standalone/adversarial.py [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+  def run_queries_from_file(self, input_csv_file, output_csv_file):
+    rows = read_tsv(input_csv_file)
+    queries = [row[0] for row in rows]
+    with open(output_csv_file, "w", newline="") as out:
+      logging.info("Writing results to: %s", output_csv_file)
+      csv_writer = csv.DictWriter(out,
+                                  fieldnames=Result.get_csv_columns(),
+                                  lineterminator="\n")
+      csv_writer.writeheader()
+      with multiprocessing.Pool() as pool:
```
Comment (pradh) [inline server/integration_tests/standalone/adversarial.py:296]: what's the parallelism?

File: server/integration_tests/standalone/adversarial.py [hunk 3]
```diff
@@ -0,0 +1,459 @@
...
+
+    resp = None
+    try:
+      resp = requests.post(self.server_url +
+                           f'/api/explore/detect-and-fulfill?q={query}',
+                           json={
+                               'contextHistory': {},
+                               'dc': '',
+                           },
+                           timeout=10)
```
Comment (pradh) [inline server/integration_tests/standalone/adversarial.py:316]: is this seconds?  set a longer timeout, like 30?  I think queries can easily take that long esp when talking to LLM

**Review by pradh — COMMENTED** [review]
Thanks!

Could you please have a README with how to run this too?

File: server/integration_tests/standalone/adversarial.py (hunk 1 above)
Comment (keyurva) [reply server/integration_tests/standalone/adversarial.py:384]: Ack. Will add this in a follow-on PR.

File: server/integration_tests/standalone/adversarial.py (hunk 2 above)
Comment (keyurva) [reply server/integration_tests/standalone/adversarial.py:293]: Number of CPUs. https://screenshot.googleplex.com/4cCRAgbBy9Y8TS2

File: server/integration_tests/standalone/adversarial.py (hunk 3 above)
Comment (keyurva) [reply server/integration_tests/standalone/adversarial.py]: Done.

File: server/integration_tests/standalone/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+# Data Commons Standalone Integration Tests
+
+The tests in this folder are standalone tests that are run against the dev or prod website deployments.
```
Comment (pradh) [inline server/integration_tests/standalone/README.md:3]: Thanks for the detailed README!  (More is great obv, the "norm" is typically a 2-line summary of what it does, and cmd lines on how to invoke)

**Review by pradh — APPROVED** [review]

**Pull request merged**